manifests/init.pp

```diff
@@ -443,6 +441,43 @@
     }
   }
 
+  if $facts['os']['family'] == 'Debian' and $facts['service_provider'] == 'systemd' {
+    include 'systemd::systemctl::daemon_reload'
+    file {[
+        '/etc/systemd/system/snmpd.service.d',
+        '/etc/systemd/system/snmptrapd.service.d/',
+      ]:
+      ensure => 'directory',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0755',
+    }
+    file {'/etc/systemd/system/snmpd.service.d/local.conf':
```

Comment:
We can use a drop-in file for this, right?
https://github.com/camptocamp/puppet-systemd#drop-in-files